_themes/cheatsheet/support.py

```diff
@@ -2,3 +2,4 @@
 from pygments.token import Keyword, Name, Comment, String, Error, \
      Number, Operator, Generic, Whitespace, Punctuation, Other, Literal
 from pygments.lexers.r import SLexer
+from pygment.lexers.stata import StataLexer
```

Comment:
should this be ``pygments``. I can't find this lexer on my default anaconda installation